tests/datasets/test_invoice.py

```
@@ -0,0 +1,35 @@
+from zipfile import ZipFile
+from tqdm import tqdm
+from fintorch.datasets.invoice import InvoiceDataset
+
+@pytest.fixture
+def invoice_dataset(tmp_path):
+    return InvoiceDataset(root=tmp_path, force_reload=True)
+
+def test_download(tmp_path):
+    print(f'tmp path:{tmp_path}')
+    # Create an instance of InvoiceDataset
+    dataset = InvoiceDataset(root=tmp_path, force_reload=True)
```

Ruff (0.8.2)

Local variable `dataset` is assigned to but never used

Remove assignment to unused variable `dataset`

(F841)